src/bin/dolos/common.rs

```diff
@@ -9,6 +9,13 @@ use tokio::task::JoinHandle;
 use tokio_util::sync::CancellationToken;
 use tracing::{debug, warn};
 use tracing_subscriber::{filter::Targets, prelude::*};
+use opentelemetry_appender_tracing::layer;
+use opentelemetry_otlp::WithExportConfig;
+use opentelemetry_otlp::{LogExporter, Protocol};
```

Comment:
Note: I only export logs and not traces (spans) or metrics just because I only needed logs to make sure it works